assemble/bin/accumulo

```diff
@@ -151,10 +151,23 @@ fi
 # Export the variables just in case they are not exported
 # This makes them available to java
 export JAVA_HOME HADOOP_PREFIX ZOOKEEPER_HOME LD_LIBRARY_PATH DYLD_LIBRARY_PATH
+
+# Strip the instance from $1
+APP=$1
+INSTANCE="1"
+if [[ "$1" =~ .*-.* ]]; then
+  APP=`echo $1 | cut -d"-" -f1`
+  INSTANCE=`echo $1 | cut -d"-" -f2`
```

Comment:
Should any validation of `INSTANCE` be done?  Could ensure its an integer >=1.  Maybe also check that its <= NUM_TSERVERS, I am not sure if check makes sense here.